applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -41,7 +41,16 @@
 -define(KEY_DELETE_AFTER_NOTIFY, <<"delete_after_notify">>).
 -define(KEY_SAVE_AFTER_NOTIFY, <<"save_after_notify">>).
 -define(KEY_FORCE_REQUIRE_PIN, <<"force_require_pin">>).
+-define(KEY_ALLOW_FF_RW, <<"allow_ff_rw">>).
+-define(KEY_SEEK_DURATION, <<"seek_duration">>).
```

Comment:
Generally speaking we try to put units of time in the names: `seek_duration_ms` in this case to ease reading.

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)